examples/azure-monitoring/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "cache" {
+	description = "The Azure Redis Cache alerts"
+
+	default = {
```

Comment:
for the purposes of the example we could hard-code these values in the resource - in particular it'd be worth provisioning the Redis Cache (and the Action Group) required for this to work end-to-end so that users can run this?